### ISSUE TITLE
`bad-test-name` lint isn't raising warnings for improperly named test objects

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#1125]
How about we add a lint that checks that every test objects is named as such (one of these):

```
can-*
cannot-*
accepts-*
rejects-*
stops-on-*
```

On any other name, the lint should raise a warning.
